src/main/resources/templates/fragments/startSection.html

```diff
@@ -0,0 +1,198 @@
+                    <p class="description text-center">Moje wypożyczone książki</p>
+                </div>
+            </div>
+            <div class="col-md-6 col-lg-3 d-flex mb-5 mb-lg-0 d-flex justify-content-center">
+                <div class="tile1">
+                    <h4 class="title"><a href=""><img class="img-fluid" src="/images/borrow.png"></a></h4>
+                    <p class="description text-center">Wypożycz książkę</p>
+                </div>
+            </div>
+            <div class="col-md-6 col-lg-3 d-flex mb-5 mb-lg-0 d-flex justify-content-center">
+                <div class="tile1">
+                    <h4 class="title"><a href=""><img class="img-fluid" src="/images/returnBook.png"></a></h4>
+                    <p class="description text-center">Zwróć książkę</p>
+                </div>
+            </div>
+            <div class="col-md-6 col-lg-3 d-flex mb-5 mb-lg-0 d-flex justify-content-center">
+                <div class="tile1">
+                    <h4 class="title"><a th:href="@{/bookList}"><img class="img-fluid" src="/images/baseBooks.png"></a></h4>
+                    <p class="description text-center">Lista książek</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<!--
+* start logoff
+-->
+
+<section th:fragment="logoff" id="start" class="d-flex align-items-center">
+    <div class="container position-relative">
+        <div class="row justify-content-center">
+            <div class="col-xl-7 col-lg-9 text-center">
+                <h1 class="library">BIBLIOTEKA</h1>
+                <h2 class="library">Witamy w naszej internetowej bibliotece</h2>
+                <form class="d-flex my-2"><input class="form-control" type="text" placeholder="Wyszukaj książkę...">
+                    <button class="btn btn-primary" type="submit">Szukaj</button>
+                </form>
+            </div>
+        </div>
+        <div class="row tiles mt-2">
+
+
+
+            <div id="carouselExampleCaptions" class="carousel carousel-dark slide">
+                <div class="carousel-indicators">
+                    <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
+                    <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" aria-label="Slide 2"></button>
+                    <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" aria-label="Slide 3"></button>
+                </div>
+                <div class="carousel-inner">
+                    <div class="carousel-item active">
+                        <img src="/images/a.png" class="d-block h-50 mx-auto" alt="a">
+                        <div class="carousel-caption d-none d-md-block">
+                            <h3>First slide label</h3>
```

Comment:
Nie potrzebnie to się wyświetla na stronie ;)

Reply:
Poprawione